pkg/fixtures/triggers/invoice.mark_uncollectible.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
This should be `marked_uncollectible` I think, right? https://docs.stripe.com/api/events/types#event_types-invoice.marked_uncollectible